### DISCUSSION TITLE
Don't show language select when there is only one language to choose from

### DISCUSSION THREAD

**Discussion by janwillemvd** [directus/directus#14236] [Feature Requests]
We think it can be useful to add Translations to a collection, even if you don't need translations yet (for instance, when you expect to support translations in the future). 

In that case, we only add one language to the languages collection of that collection and add all the fields. Obviously, you don't need the language selectbox when there's only one language to  choose from:
![Screenshot 2022-07-01 at 08 58 00](https://user-images.githubusercontent.com/10515908/176841347-0f9ef258-8dbf-434c-8322-f53ef1d214d2.png)

Then, in the future, when you add more languages, the selectbox will appear again.

I'm happy to create a PR for this?

**Comment by rijkvanzanten** [comment]
So you'd effectively hide the two icons on the right hand side, or hide the whole select box completely?

**Reply by janwillemvd** [reply]
My idea was to hide the whole selectbox completely (but keep the small language pills on the individual fields to still show the current language).

**Reply by rijkvanzanten** [reply]
Makes sense to me!

**Reply by rijkvanzanten** [reply]
Wanna take a swing at the PR?

**Comment by rijkvanzanten** [comment]
✅ Idea is accepted. Implementation will be tracked in https://github.com/directus/directus/issues/14243 🎉. 

Any discussion around the implementation details of this feature can be found over there.

**Reply by janwillemvd** [reply]
Nice, thanks, PR is on its way! :)
